README.md

```diff
@@ -77,66 +77,226 @@ The authentication flow is as follows:
 
 If you already have rbenv installed and configured on your system, the correct Ruby version will be set, when you access the app directory.
 
-Some dependencies (eg CocoaPods) are installed via [bundler](https://bundler.io/).
+To install, follow the steps described below.
 
-#### React Native
+##### Installing `rbenv` on macOS
 
-Follow the tutorial [Building Projects with Native Code](https://facebook.github.io/react-native/docs/getting-started.html) for your operating system.
+You should already have installed `brew` so use:
 
-If you have a macOS system, you can follow both the tutorial for iOS and for Android. If you have a Linux or Windows system, you need only to install the development environment for Android.
+```
+brew install rbenv
+```
 
-### Building and launching on the simulator
+Brew installs `rbenv` in the path so no more steps are needed.
 
-#### Dependencies
+##### Installing `rbenv` on Linux 
 
-First we install the libraries used by the project:
+This is the generic installation procedure for Linux that should work on many distributions. The procedure has been tested on Ubuntu Linux. 
 
 ```
-$ bundle install
-$ yarn install
-$ cd ios
-$ pod install
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+mkdir ~/.rbenv/plugins
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
 ```
 
-#### Generating API definitions and translations
+Add `rbenv` to the PATH as follows then reload the intialization file
 
-The second step is to generate the definitions from the openapi specs and from the YAML translations:
+```
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >>~/.bashrc
+source ~/.bashrc
+```
+
+Verify you have installed it correctly with the command `which rbenv`.
+
+#### Completing and verifying configuration
+
+Either on Mac or Linux you need to add to your shell the initialization command thjen reload the configuration:
 
 ```
-$ yarn generate:all
+echo 'eval "$(rbenv init -)"' >>~/.bashrc
+source ~/.bashrc
+```
+
+(if you use a different shell than bash you may need to adapt the command to your shell initialization files).
+
+
+Before you can install your version of Ruby, you need a C compiler and some libraries. On Ubuntu or Debian based systems use:
+
+```
+sudo apt-get update
+sudo apt-get install build-essential libssl-dev libreadline-dev zlib1g-dev
+ ```
+
+Now you can install your version of `ruby` using `rbenv` (replace `<work-dir>` with your actual work directory)
+
+```
+cd <work-dir>/io-app
+rbenv install
+```
+
+You should verify that the output of the `rbenv version` command and the content of the file `.ruby-version` are the same:
+
+For example (replace `<work-dir>` with your actual work directory):
+
+```
+$ cd <work-dir>/io-app
+$ rbenv version
+2.4.2 (set by <work-dir>/io-app/.ruby-version)
+$ ruby -v
+ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-linux]
+$ cat .ruby-version
+2.4.2
+```
+
+#### Install bundler and cocoa
```

Comment:
Cocoapods must not be installed using `gem install ....` because it will be installed by bundler on `bundle install` reading the `Gemfile`.

Reply:
yes you right, I checked, it is redundant